hosting/src/app/(protected)/layout.tsx

```diff
@@ -2,11 +2,19 @@
 
 import CmsLayout from "@/components/Layouts/CmsLayout";
 import React from "react";
+import {useAuthentication} from "@/hooks/useAuthentication";
+import {redirect} from "next/navigation";
 
 interface ProtectedLayoutProps {
   children: React.ReactNode;
 }
 
 export default function ProtectedLayout({children}: ProtectedLayoutProps) {
+  const {isSignedIn} = useAuthentication();
+
+  if (isSignedIn === false) {
+    redirect("/auth");
```

Comment:
Could you change this to `redirect ("/");` instead to make it consistent with how we want landing page to work 🙏

Reply:
I've updated on this commit https://github.com/oddbit/tanam/pull/373/commits/03f54385681bb11d8553dbb9f7bbab71689e357b ☺️